Documentation/contributor-guide/community-membership.md

```diff
@@ -115,6 +115,14 @@ depend on to make decisions in the best interest of the project in a consistent
 - Subscribed to [etcd-maintainers@googlegroups.com]
 - Elected by supermajority of active maintainers.
 
+### Onboarding process
+
+- Open a PR and add an entry to the [MAINTAINERS] file.
+- Join to [etcd-maintainers@googlegroups.com] and [etcd-maintainers-private@googlegroups.com]
```

Comment:
This is already mentioned as a requirement

Reply:
yeah, I'll remove it